datalad/distributed/ora_remote.py

```diff
@@ -416,15 +428,19 @@ def get(self, src, dst, progress_cb):
         # actual get (that is 'cat').
         # Therefore check beforehand.
         if not self.exists(src):
-            raise RIARemoteError("annex object {src} does not exist.".format(src=src))
+            raise RIARemoteError("annex object {src} does not exist."
+                                 "".format(src=src))
```

Comment:
`s/"".format/.format/` would also work, though I'm fine with `"".format()` if you prefer it style-wise.

Reply:
Yeah, to me it's more obvious that way when skimming through code.